lib/query/compiler.js

```diff
@@ -90,6 +93,14 @@ class QueryCompiler {
       assign(query, val);
     }
 
+    if (isString(query.sql) && Array.isArray(this._tokenTransformers)) {
```

Comment:
Probably case when query doesn't have `sql` is less frequent than when tokenTransformers are not set? If that's the case, probably it would make sense to swap conditions.

Reply:
Yes, I agree. Updated.

Reply:
Yeah I agree. Update the order of checks.